tests/upgrades/test_classparameter.py

```diff
@@ -19,7 +19,21 @@
 import json
 
 import pytest
-from nailgun import entities
+
+from robottelo.utils.installer import InstallerCommand
+
+common_opts = {
+    'foreman-proxy-puppetca': 'true',
+    'foreman-proxy-content-puppet': 'true',
+    'foreman-proxy-puppet': 'true',
+    'puppet-server': 'true',
+    'puppet-server-foreman-ssl-ca': '/etc/pki/katello/puppet/puppet_client_ca.crt',
+    'puppet-server-foreman-ssl-cert': '/etc/pki/katello/puppet/puppet_client.crt',
+    'puppet-server-foreman-ssl-key': '/etc/pki/katello/puppet/puppet_client.key',
+    # Options for puppetbootstrap test
+    'foreman-proxy-templates': 'true',
+    'foreman-proxy-http': 'true',
+}
```

Comment:
I see this is a duplicate of what is in `pytest_fixtures/component/puppet.py`. Can we store the dictionary in some common place instead of duplicating it?